packages/app/src/apis.ts

```diff
@@ -114,7 +114,10 @@ export const apis = (config: ConfigApi) => {
 
   builder.add(featureFlagsApiRef, new FeatureFlags());
 
-  builder.add(lighthouseApiRef, new LighthouseRestApi('http://localhost:3003'));
+  builder.add(
+    lighthouseApiRef,
+    new LighthouseRestApi(config.getString('lighthouse.baseUrl')),
```

Comment:
I'd vote for a `LighthouseRestApi.fromConfig` variant instead. Not too excited about the idea of these config paths diverging for different apps. Better code it into the plugin itself imo.

Reply:
@Rugvip - Let me know if this was what you had in mind.

Reply:
@andrewthauer, yep, exactly that, thanks! :grin: